Cargo.toml

```
@@ -20,7 +20,7 @@ exclude = [
 
 [workspace.package]
 version = "22.0.0-rc.1"
-rust-version = "1.81.0"
+rust-version = "1.82.0"
```

One downside of this that just occurred to me, is that the soroban-sdk only supports up to Rust 1.81, but this change makes it so you need Rust 1.82 to do a cargo install of the stellar-cli, when that version isn't actually required. That will mean devs would need both versions to install from source, which some do.

See https://github.com/stellar/rs-soroban-sdk/blob/9324422a11daa99fdf31d420bb4072dbd875de60/soroban-sdk/build.rs

Can we omit this part of the change?

```suggestion
rust-version = "1.81.0"
```